pyms/IonChromatogram.py

```diff
@@ -302,7 +300,7 @@ def __calc_time_step(self) -> float:
 
 		return time_step
 
-	def write(self, file_name : Union[str, pathlib.Path], minutes: bool = False, formatting = True):
+	def write(self, file_name : Union[str, pathlib.Path], minutes: bool = False, formatting = bool):
```

Comment:
Sorry, the type of formatting should be `bool`; the value should still be `True`.